lib/app.rb

```diff
@@ -106,10 +118,16 @@ def repo_defined?(source, org, name)
 scheduler.every('1h') do
   worker_fetch_all(repositories.values)
 end
+scheduler.every('1h5m') do
+  worker_all_badges(repositories.values)
+end
 
 puts "Running Workers a first time..."
 include RubyDepsHelpers
 worker_fetch_all(repositories.values)
 
+sleep 5
```

Comment:
Why sleep?

Reply:
The BadgeWorker is so lightning fast that it would finish before the DependencyWorkers were done (even though those threads are being fired up earlier)... yielding empty badges. :) The 5 is rather arbitrary, don't know if there's a more sensible way of doing that.

Reply:
Either the badge workers should wait for data to be stored (either by polling the store every so often, or, preferably, through some kind of callback), or the badge job should be scheduled/fired up by the dependency workers.